docs/source/en/package_reference/inference_endpoints.md

```diff
@@ -0,0 +1,46 @@
+# Inference Endpoints
+
+Inference Endpoints offers a secure production solution to easily deploy any `transformers`, `sentence-transformers` and `diffuser` models from the Hub on a dedicated and autoscaling infrastructure managed by Hugging Face.
+
+A Hugging Face Endpoint is built from a [Hugging Face Model Repository](https://huggingface.co/models). When an Endpoint is created, the service creates image artifacts that are either built from the model you select or a custom-provided container image. The image artifacts are completely decoupled from the Hugging Face Hub source repositories to ensure the highest security and reliability levels.
+
+🤗 Inference Endpoints support all of the `transformers`, `sentence-transformers` and `diffuser` tasks as well as custom tasks not supported by `transformers` yet like speaker diarization and diffusion.
+
+In addition, 🤗 Inference Endpoints gives you the option to use a custom container image managed on an external service, for instance, [Docker Hub](https://hub.docker.com/), [AWS ECR](https://aws.amazon.com/fr/ecr/?nc1=h_ls), [Azure ACR](https://azure.microsoft.com/de-de/products/container-registry/), or [Google GCR](https://cloud.google.com/artifact-registry?hl=de).
+
+<Tip>
+
+This page is only a package reference about `huggingface_hub`'s integration with Inference Endpoints. For more information about Inference Endpoints product, check out its [official documentation](https://huggingface.co/docs/inference-endpoints/index).
+
+</Tip>
+
+## API
```

Comment:
I think we can remove this section header since its clear we're already here on the API page.